app/src/screens/StakingTokens.js

```diff
@@ -0,0 +1,218 @@
+            css={`
+              margin-bottom: 0px;
+            `}
+          >
+            <div
+              css={`
+                ${textStyle('title2')};
+              `}
+            >
+              {`${
+                stakeToken.balance
+                  ? formatTokenAmount(
+                      parseInt(stakeToken.balance),
+                      parseInt(stakeToken.tokenDecimals)
+                    )
+                  : '-'
+              } ${stakeToken.tokenSymbol}`}
+            </div>
+            <div
+              css={`
+                ${textStyle('body4')};
+                color: ${theme.contentSecondary};
+              `}
+            >
+              {stakeToken.balance
+                ? (parseInt(stakeToken.balance) /
+                    parseInt(stakeToken.tokenSupply)) *
+                  100
+                : '-'}
+              % of total tokens
+            </div>
+          </Field>
+        </div>
+      </div>
+      {myStakes && myStakes.length > 0 && (
+        <div
+          css={`
+            border-bottom: 1px solid ${theme.border};
+          `}
+        >
+          <div
+            css={`
+              padding: ${3 * GU}px;
+            `}
+          >
+            <Field
+              label="Your active tokens"
+              css={`
+                margin-bottom: 0px;
+              `}
+            >
+              <div
+                css={`
+                  ${textStyle('title2')};
+                  color: ${theme.infoSurfaceContent};
+                `}
+              >
+                {`${
+                  myActiveTokens
+                    ? formatTokenAmount(
+                        parseInt(myActiveTokens),
+                        parseInt(stakeToken.tokenDecimals)
+                      )
+                    : '-'
+                } ${stakeToken.tokenSymbol}`}
+              </div>
+              <div
+                css={`
+                  ${textStyle('body4')};
+                  color: ${theme.contentSecondary};
+                `}
+              >
+                Total Active Tokens:{' '}
+                {formatTokenAmount(
+                  parseInt(totalActiveTokens),
+                  parseInt(stakeToken.tokenDecimals)
+                )}{' '}
+                {stakeToken.tokenSymbol}
+              </div>
+            </Field>
+
+            <Field
+              label="SUPPORTED PROPOSALS"
+              css={`
+                margin-top: ${3 * GU}px;
+              `}
+            >
+              <div
+                css={`
+                  margin-top: ${2 * GU}px;
+                `}
+              >
+                <Distribution
+                  heading="Your active token distribution"
+                  items={stakes}
+                  renderLegendItem={({ item }) => {
+                    return (
+                      <div
+                        css={`
+                          background: #daeaef;
```

Comment:
```suggestion
                          background: ${theme.badge};
```

Reply:
is not the same color `badge: '#EBFAFD' on theme-light

Reply:
Is there a reason we are using a new color instead of just the `aragon-ui` colors?

Reply:
no idea haha